Jenkinsfile

```diff
@@ -61,7 +61,14 @@ try {
 variantList.each {
     def list = it.split(":")
     variantMap["${list[0]}"] = {
-        buildOnYoctoNode(list[0], list[1])
+
+        // TODO get these values from variantMap (or similar).
```

Comment:
I assume this also needs to be fixed before merging.

Today `VARIANT_LIST` is used. Syntax is a list of items of format `<variant>:<image>`

Any ideas of the new format?
`<machine>:[<feature>:][<feature>:][<feature>:]<image>`

Reply:
I like the format. however : is not a good separator since it is used by bitbake multiconfig and will therefore cause issues when building using meta-acrn.

So yes to the format with a bitbake safe separator such as "_" or "|"

Reply:
I prefer `|` or `#`, e.g.
`rcarh3#qtas#acrn`
`rcarh3|qtas|acrn`

The `#` might be more readable than `|`

Reply:
I vote for "|"